src/yb/master/backfill_index.cc

```diff
@@ -678,6 +667,26 @@ BackfillTable::BackfillTable(
   done_.store(false, std::memory_order_release);
 }
 
+std::string BackfillTable::RetrieveIndexNames(const std::unordered_set<std::string>& requested_index_ids) {
+  std::ostringstream out;
+  out << "{ ";
+  bool first = true;
+  for (const auto& index_id : requested_index_ids) {
+    const auto table_info = master_->catalog_manager()->GetTableInfo(index_id);
+    if (!table_info) {
+       continue;
+    }
```

Comment:
Should this missing table info log warning or do something more severe?  For example, `LOG(WARNING) << "foo";`